packages/nextjs/app/builders/0x822e7a7209d6C3Da71979AD443D1D3e81E78e2FD/page.tsx

```diff
@@ -0,0 +1,50 @@
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const BillPage: NextPage = () => {
+  return (
+    <div className="flex items-center flex-col flex-grow justify-center">
+      <div className="flex gap-10 bg-accent-content/30 rounded-lg">
```

Comment:
You could add something like: `flex flex-col md:flex-row` to make it look good on mobile too!

Reply:
Good call this is much better.